Homebrew/Archetypes/Bard/Bard (Discord).xml

```diff
@@ -0,0 +1,43 @@
+				<text>	As part of this bonus action, you can expend one of your uses of Bardic Inspiration, rolling a Bardic Inspiration die and adding the number rolled to the save DC. You can choose to use this feature after the creature makes its roll but before the DM announces if it succeeds or fails.</text>
+			</feature>
+		</autolevel>
+		<autolevel level="6">
+			<feature optional="YES">
+				<name>College of Discord: Encore</name>
+				<text>At 6th level, every success fuels your performance. When you succeed on a Strength, Dexterity, or Constitution saving throw in combat, you gain advantage on your next skill check, saving throw, or attack roll until the end of your next turn.</text>
+				<text>	Additionally, your attacks with your musical instruments count as magical for the purpose of overcoming resistance and immunity to nonmagical attacks and damage.</text>
+			</feature>
+		</autolevel>
+		<autolevel level="14">
+			<feature optional="YES">
+				<name>College of Discord: Pandemonium</name>
+				<text>At 14th level, you learn to provoke those the area around you by amplifying the clash of arms and  ailing bodies, trumpeting your allies to hasten attack while sending your enemies into dysphonic chaos that is audible with 300 feet of you. Using your action to activate this feature, you create a hyper-sonic field that reverberates out from you in a 30-foot cube which requires concentration and lasts for 1 minute.</text>
+				<text>	When the field appears, friendly creatures within the field experience a haste-like effect and gain a +2 bonus to AC, have advantage on Dexterity saving throws, and gain an additional action on each turn. That action can be used only to take the Attack (one weapon attack only), Dash, Disengage, Hide, or Use an Object action. A creature can only benefit from one such effect at a time. While you concentrate on this effect, your speed is 0, and you can only make melee attacks against creatures within 5 feet of you. If you have a fly speed, or are held in place my magical means, you can hover in place.</text>
+				<text>	Additionally, when a hostile creature starts its turn within the field, or moves there for the first time, must make a Wisdom saving throw or be affected for the duration. Affected creatures' speed is halved, takes a –2 penalty to AC and Dexterity saving throws, and can't use reactions. On an affected creature's turn, it can use either an action or a bonus action, not both. Regardless of the creature's abilities or magic items, it can't make more than one melee or ranged attack during its turn.</text>
+				<text>	If the creature attempts to cast a spell with a casting time of 1 action, roll a d20. On an 11 or higher, the spell doesn't take e ect until the creature's next turn, and the creature must use its action on that turn to complete the spell. If it can't, the spell is wasted. If a creature affected by this feature exits the field, the effect ends for it at the start of its next turn.</text>
```

Comment:
```suggestion
				<text>	If the creature attempts to cast a spell with a casting time of 1 action, roll a d20. On an 11 or higher, the spell doesn't take effect until the creature's next turn, and the creature must use its action on that turn to complete the spell. If it can't, the spell is wasted. If a creature affected by this feature exits the field, the effect ends for it at the start of its next turn.</text>
```